crawler/crawler/cloneRepository.go

```diff
@@ -28,22 +28,14 @@ func CloneRepository(domain Domain, hostname, name, gitURL, index string) error
 
 	// If folder already exists it will do a fetch instead of a clone.
 	if _, err := os.Stat(path); !os.IsNotExist(err) {
-		//	Command is: git fetch --all
 		out, err := exec.Command("git", "-C", path, "fetch", "--all").CombinedOutput() // nolint: gas
```

Comment:
Does `fetch` respect `blob:none` in subsequent calls if a repo was cloned with `--filter=blob:none`?

Reply:
Yes. The information about the filter is written in `config`, e.g.:

```console
$ git clone  --filter=blob:none --mirror https://github.com/italia/developers-italia-backend
$ cat developers-italia-backend.git/config                                                                                                                                                                                                   
[core]
	repositoryformatversion = 1
	filemode = true
	bare = true
	ignorecase = true
	precomposeunicode = true
[remote "origin"]
	url = https://github.com/italia/developers-italia-backend
	fetch = +refs/*:refs/*
	mirror = true
	promisor = true
	partialclonefilter = blob:none
```